tedana/decomposition/eigendecomp.py

```diff
@@ -143,21 +143,22 @@ def tedpca(catd, OCcatd, combmode, mask, t2s, t2sG, stabilize,
             from sklearn.decomposition import PCA
             ppca = PCA(n_components='mle', svd_solver='full')
             ppca.fit(dz)
-            v = ppca.components_
-            s = ppca.explained_variance_
-            u = np.dot(np.dot(dz, v.T), np.diag(1. / s))
+            comp_ts = ppca.components_
+            varex = ppca.explained_variance_
+            voxel_comp_weights = np.dot(np.dot(dz, comp_ts.T), np.diag(1. / varex))
         else:
-            u, s, v = np.linalg.svd(dz, full_matrices=0)
+            voxel_comp_weights, varex, comp_ts = np.linalg.svd(dz, full_matrices=0)
 
         # actual variance explained (normalized)
-        sp = s / s.sum()
```

Comment:
As an alternative, maybe what we should be doing is -- starting here-- break the above into a separate function, returning `u`, `s`, and `v`, so they can be imported as more meaningful variable names where they're actually used.

Reply:
:+1: for moving this into its own function!

Reply:
Let me know what you both think of the new function.

Reply:
:+1: looks good to me.